### ISSUE TITLE
Links to chapters in custom syllabus should open in a new tab

### ISSUE THREAD

**Issue by veenaiyuri** [Qiskit/platypus#2112]
### Description of issue

This issue relates to #351 

The links to each chapter/problemset that are added using the [custom syllabus feature](https://learn.qiskit.org/syllabus/create) should have anchor tags so that they open in new tabs/windows. 
Students and educators who are utilizing the custom syllabus want to keep their syllabus page visible while clicking on the links to the chapters in the [texbook](https://qiskit.org/learn/). 

Given the transition, I understand that the priority for fixing this may be low. That being said, as we have multiple universities that are currently utilizing our custom syllabus feature to deliver their courses in 2023, this fix would be much appreciated.

**Comment by vabarbosa** [comment]
This repository is no longer maintained. It has been superseded by [IBM Quantum Learning](https://learning.quantum.ibm.com)
